Modules/Devices/src/Devices.Application/Clients/Queries/GetClient/Handler.cs

```diff
@@ -17,6 +17,6 @@ public async Task<ClientDTO> Handle(GetClientQuery request, CancellationToken ca
     {
         var client = await _oAuthClientsRepository.Find(request.Id, cancellationToken) ?? throw new NotFoundException(nameof(OAuthClient));
 
-        return new ClientDTO(client.ClientId, client.DisplayName, client.DefaultTier);
+        return new ClientDTO(client.ClientId, client.DisplayName, client.DefaultTier, client.CreatedAt);
```

Comment:
The DTO is currently setup like this:
![image](https://github.com/nmshd/backbone/assets/118175875/042d4a94-dc9f-44d8-aa08-61333ace6fe9)

So for this case, I think the best approach would be to use the mapper, right?
![image](https://github.com/nmshd/backbone/assets/118175875/c371a0c6-02eb-4228-9c68-47a4826a0a0d)

Reply:
If we also decide not to use the ClientDTO constructor anymore, we would have to change it to something like this, because of the TierId -> string conversion for the DefaultTier:

![image](https://github.com/nmshd/backbone/assets/118175875/b8a1159b-18a0-4ce8-8928-6c4eee8d60e8)

Reply:
I'm still not sure which way I prefer to go (AutoMapper vs. custom mapping in the constructor).

But until I have decided on this topic, I'm fine with using AutoMapper.